README.md

```diff
@@ -36,17 +34,7 @@ This repository contains submodule tesseract_ext so use the *--recursive* flag a
 
 ## Building with Clang-Tidy Enabled
 
-Must pass the -DTESSERACT_ENABLE_CLANG_TIDY=ON to cmake when building. This is automatically enabled if cmake argument -DTESSERACT_ENABLE_TESTING=ON is passed.
-
-## Building Tesseract Tests
-
-Must pass the -DTESSERACT_ENABLE_TESTING=ON to cmake when wanting to build tests. This automatically enables clang tidy.
```

Comment:
Should we specify how to build tesseract_ros tests or does catkin run_tests rebuild with CATKIN_ENABLE_TESTING=ON already?

Reply:
catkin run_tests will handle building tests for tesseract_ros